restcomm/restcomm.mgcp/src/main/java/org/restcomm/connect/mgcp/AsrSignal.java

```diff
@@ -57,6 +60,9 @@ public AsrSignal(String driver, String lang, List<URI> initialPrompts, String en
         this.timeAfterSpeech = timeAfterSpeech;
         this.hotWords = hotWords;
         this.lang = lang;
+        this.input = input;
+        this.minNumber = numberOfDigits;
+        this.maxNumber = numberOfDigits;
```

Comment:
thanks @YevgenL as i suggested in my initial comment, can we add this explanation as comment so its helpful for code reader?